src/test/functional/manage-users-test.ts

```diff
@@ -11,23 +10,23 @@ import {
 import '../../main/utils/utils';
 
 Feature('Manage Existing User');
-import {config as testConfig, testAccounts} from '../config';
+import {config as testConfig} from '../config';
 import * as Assert from 'assert';
 import {randomData} from './shared/random-data';
 import {convertISODateTimeToUTCFormat} from '../../main/utils/utils';
 
-const TEST_SUITE_PREFIX = 'TEST_MANAGE_USER';
-const credentials = new DataTable(['email', 'password']);
-credentials.add([testConfig.SMOKE_TEST_USER_USERNAME, testConfig.SMOKE_TEST_USER_PASSWORD]);
-credentials.add([testAccounts.superUser.email, testConfig.PASSWORD]);
-credentials.add([testAccounts.adminUser.email, testConfig.PASSWORD]);
-
-Data(credentials).Scenario('I should be able to see the active status of an user', async ({I, current}) => {
+const dashboardUserEMAIL = testConfig.TEST_SUITE_PREFIX + randomData.getRandomEmailAddress();
+BeforeSuite(async () => {
+  const dashboardUser = await createUserWithRoles(dashboardUserEMAIL, testConfig.PASSWORD, testConfig.USER_FIRSTNAME, []);
+  console.log(dashboardUser.id, dashboardUser.email);
```

Comment:
```suggestion
```
I'm presuming this is just leftover from dev? :)

Reply:
Yes, fixed